plugins/GSdx/GSRendererDX9.cpp

```diff
@@ -58,11 +58,17 @@ bool GSRendererDX9::CreateDevice(GSDevice* dev)
 
 void GSRendererDX9::EmulateTextureShuffleAndFbmask()
 {
-	if (m_texture_shuffle) {
+	if (m_texture_shuffle)
+	{
+		m_ps_sel.dfmt = 0;
```

Comment:
Is this needed if the draw is going to be skipped?

Reply:
So why do you think the alpha mask code might run?

Reply:
Yeah was useless like you said so updated.